backend/push.js

```diff
@@ -239,6 +239,7 @@ export const postEvent = async (subscription: Object, event: ?string): Promise<v
   })
 
   if (!req.ok) {
+    console.debug('Error sending push notification', subscription.id, req.status)
```

Comment:
Should this be console.debug or something higher like log or info?

Reply:
Maybe. I thought `debug` is appropriate since it's by definition a user-defined endpoint. Nothing stops a user from reporting an endpoint like `https://www.bankofamerica.com/`, and since it's a third-party endpoint, we also can't control the SLA.